data/crac/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/json/serializers/range_action/CountertradingSerializer.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.farao_community.farao.data.crac_impl.json.serializers.range_action;
+
+import com.farao_community.farao.data.crac_impl.remedial_action.range_action.Countertrading;
+
+/**
+ * @author Joris Mancini {@literal <joris.mancini at rte-france.com>}
+ */
+public class CountertradingSerializer extends RangeActionSerializer<Countertrading> {
```

Comment:
Can we remove this file ? (the class is not used)

Reply:
ok